.storybook/preview.js

```diff
@@ -1,3 +1,7 @@
+// this code was adapted from examples found in the following issue on
+// how to sort stories, from the Storybook GitHub repository issues:
+// https://github.com/storybookjs/storybook/issues/548#issuecomment-660354226
+
 const hasKey = (obj, key) => Object.prototype.hasOwnProperty.call(obj, key)
 const compareAlphabetical = (a, b) => a.localeCompare(b, { numeric: true })
```

Comment:
Yeah, looks like there are a lot of issues surrounding sorting for Storybook. Adding this link just in case it will be helpful in the future: 
https://github.com/storybookjs/storybook/issues/548